test/dynamo/test_higher_order_ops.py

```diff
@@ -1631,31 +1637,26 @@ def f(x):
 
         self.assertEqual(result, x.sin() + x.cos())
         self.assertEqual(cnt.frame_count, 1)
+        self.assertEqual(len(counters["graph_break"]), 0)
         self.assertEqual(len(backend.graphs), 1)
```

Comment:
can we do a self.assertExpectedInline on backend.graphs[0]? We should see that the wrap's outputs are flattened